src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+
+    public Figure getDefaultFigure() {
+        return new Circle("white", 10);
+    }
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        switch (figureNumber) {
+            case 1:
+                return new Square(randomColor.getRandomColor(), random.nextInt(101));
```

Comment:
It would be better to move the creation of each shape into a separate private method, like createRandomSquare()